src/OpenSage.Game/Logic/Player.cs

```diff
@@ -139,15 +139,15 @@ internal static IEnumerable<Player> FromMapData(Data.Map.Player[] mapPlayers, As
             return players.Values;
         }
 
-        public static Player FromTemplate(PlayerTemplate template, PlayerSetting? setting = null)
+        public static Player FromTemplate(GameData gameData, PlayerTemplate template, PlayerSetting? setting = null)
         {
             // TODO: Use rest of the properties from the template
             return new Player(template)
             {
                 Side = template.Side,
                 Name = setting == null ? template.Name : setting?.Name,
                 DisplayName = template.DisplayName.Translate(),
-                Money = (uint) template.StartMoney,
+                Money = (uint)(template.StartMoney + gameData.DefaultStartingCash),
```

Comment:
Every player starts with 10k$, as defined inside the gameData

Reply:
The actual starting cash is specified by skirmish settings - I guess `gameData.DefaultStartingCash` should be used for pre-selecting it as an option. This is good enough for now, but not exactly correct behaviour.